### DISCUSSION TITLE
Apple TV app

### DISCUSSION THREAD

**Discussion by skywalker123p** [ente-io/ente#883] [Enhancements]
**Describe the feature**
Apple TV app is needed.

its very important to have Apple TV app support which can be good for replacing synology photos app. thanks.

**Comment by jberstler** [comment]
Supposing that someone like myself would like to start working on this, is there backend API documentation to help me figure out how to start talking to the photos app from a remote client?

**Reply by vishnukvmd** [reply]
Hey, not right now. It is possible to infer the HTTP API from our [request handlers](https://github.com/ente-io/ente/blob/7be2c66fb69517eb1a2f67b0d94cc72817687df9/server/cmd/museum/main.go#L396). What will be harder will be to perform the necessary cryptography over the requests / responses. Our [CLI](https://github.com/ente-io/ente/tree/main/cli#readme) might be a good reference point for this.

**Comment by GETandSELECT** [comment]
Thank you very much for the great work with Ente photos (the 2FA Authenticator is also an excellent app).

The only reason I haven't (yet) bought a subscription is because I can't view the photos on the big screen (TV).
I read on Reddit that an AppleTV and Google TV app is in the works? What does it look like?

**Reply by vishnukvmd** [reply]
We're initially adding a feature to cast a slideshow to any big screen that can open a browser. Once that's shipped, we will add custom implementations for both Chromecast and Apple TV.

**Reply by synologyy** [reply]
Is There any Update for appletv?

**Comment by chrssndrs** [comment]
Id love an Apple TV as well. View only is good enough to start.

Or.. maybe some adjust to the Ente app such that it behave the same way the iOS Photos app does while screen mirroring. So, photos are always displayed full screen.
